TimberAPIExample/Examples/EntityLinkerExample/UI/LinkerFragment.cs

```diff
@@ -149,9 +150,9 @@ public virtual void AddAllLinkViews()
                 };
                 view.Q<Button>("RemoveLinkButton").clicked += delegate
                 {
-                    Console.WriteLine($"link: {link}");
-                    Console.WriteLine($"linker: {link.Linker}");
-                    Console.WriteLine($"linkee: {link.Linkee}");
+                    Log.LogInfo($"link: {link}");
```

Comment:
These logs were actually just for debugging while developing. They can be removed, I seem to have forgotten